File: src/main/java/net/igneo/icv/entity/helmet/divineLightningRod/DivineLightningRodRenderer.java
package net.igneo.icv.entity.helmet.divineLightningRod;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class DivineLightningRodRenderer extends GeoEntityRenderer<DivineLightningRodEntity> {
    public DivineLightningRodRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new DivineLightningRodModel());
    }
}

File: src/main/java/net/igneo/icv/entity/helmet/divineLightningRod/DivineLightningRodRenderer.kt
package net.igneo.icv.entity.helmet.divineLightningRod

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class DivineLightningRodRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<DivineLightningRodEntity?>(renderManager, DivineLightningRodModel())
